tests/unit/test_record_checker.py

```diff
@@ -172,5 +172,5 @@ def mock_resolve_query(mocker, get_mock_coro, record_checker_instance):
     with pytest.raises(exceptions.InvalidDNSHost) as e:
         record_checker.RecordChecker(dns_ip)
 
-    expected_msg = 'RecordChecker got invalid DNS server ip: 8.855.0.5'
+    expected_msg = 'RecordChecker got invalid DNS server IP: 8.855.0.5'
```

Comment:
This is fine, but since you're adding '.'s everywhere else, how about here, too?